packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -113,6 +128,22 @@ export default class Menu extends Component {
     return signOut()
   }
 
+  _subscribeMissingPatches = () => {
+    const unsubs = map(this.props.hosts, host =>
+      subscribeHostMissingPatches(host, patches => {
+        this.setState(({ nMissingPatches = 0 }) => ({
+          nMissingPatches: nMissingPatches + patches.length,
```

Comment:
So this will keep increasing every time it receives a result from a subscription?

Reply:
Yes. So, the calculation is not correct.